### DISCUSSION TITLE
Problema con Vue

### DISCUSSION THREAD

**Discussion by rod641916** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#526] [Q&A]
Buenas tardes espero y me puedan ayudar a solucionar este error con Vue que no logro que se solucione 
![Captura](https://user-images.githubusercontent.com/95432602/160194830-94ee149c-587d-4ce5-ab3a-928bdfae0b69.PNG)

**Comment by doguedogue** [comment]
Cada uno de los directorios dentro de 05-VUEJS en programas es un proyecto.

Ya que lo descargaste todo, posicionate en la ruta del proyecto que quieres y escribe este comando:
`npm install`

![image](https://user-images.githubusercontent.com/23409026/160201797-3cbc2e44-a8b2-4e1e-b6ef-0acd6d0a75df.png)

![image](https://user-images.githubusercontent.com/23409026/160201815-b5733e83-1188-46cd-8a6b-fbe5b30bb12b.png)
